docs/Permissions/Onchain-Permissioning.md

```diff
@@ -0,0 +1,191 @@
+description: Onchain Permissioning
+<!--- END of page meta data -->
+
+# Onchain Permissioning 
+
+Onchain permissioning uses smart contracts to store and maintain the node whitelist. Using onchain permissioning
+enables all nodes to read the node whitelist from one location. 
+                                                                       
+!!! note
+    Pantheon supports onchain permissioning for nodes. Onchain permissioning for accounts will be available 
+    in a future Pantheon release. 
+
+Onchain permissioning uses the following smart contracts: 
+
+* Ingress - a simple contract functioning as a gateway to the Rules contract. The Ingress contract is deployed 
+to a static address. 
+
+* Rules - stores the node whitelist and list of admin accounts. 
```

Comment:
the Rules and Admin can be (and are likely to be very soon) separate contracts - don't know if you want to be more specific here

Reply:
Rules - stores the node whitelist.
Admin - stores the list of admin accounts.

Reply:
Fixed